source/reference/json/data-federation-config-file.txt

```diff
@@ -0,0 +1,103 @@
+.. _atlas-cli-data-federation-config-file:
+
+========================================
+{+adf+} Configuration File
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can use an {+adf+} configuration file to specify the required 
+settings for :ref:`creating a federated database
```

Comment:
when I click in this link it takes me to https://www.mongodb.com/docs/atlas/data-federation/#std-label-atlas-data-federation is this expected? looking at other json files, it usually forwards me to the command that uses the json file e.g. [backup](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-31036/reference/json/cloud-backup-schedule-config-file/) takes me to [command](https://www.mongodb.com/docs/atlas/cli/stable/command/atlas-backups-schedule-update/)

Reply:
Another great catch! I updated the link to point to the command.